components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -0,0 +1,41 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialSnackbar.h"
+
+@interface MDCSnackbarMessageViewTests : XCTestCase
+
+@end
+
+@implementation MDCSnackbarMessageViewTests
+
+- (void)testDefaultColors {
+  // Given
+  MDCSnackbarMessageView *messageView = [[MDCSnackbarMessageView alloc] init];
+
+  // Then
+  XCTAssertEqualObjects(messageView.snackbarMessageViewBackgroundColor,
+                        [UIColor colorWithRed:(float)(0x32 / 255.0)
```

Comment:
Super-nit: (CGFloat)

Reply:
I chose `float` here because `double` will fail on 64-bit devices. The function that creates these colors uses float values.  I didn't want to change that function for this commit.

```objectivec
static inline UIColor *MDCRGBAColor(uint8_t r, uint8_t g, uint8_t b, float a) {
  return [UIColor colorWithRed:(r) / 255.0f green:(g) / 255.0f blue:(b) / 255.0f alpha:(a)];
}
```